arrows/vtk/applets/fuse_depth.cxx

```diff
@@ -366,10 +396,29 @@ class fuse_depth::priv
         mesh_writer->AddInputDataObject(isosurface_mesh);
         mesh_writer->Write();
       }
+      else if ( extension == ".las" )
+      {
+        auto lgcs = vital::local_geo_cs();
+        read_local_geo_cs_from_file(lgcs, input_geo_origin_file);
+
+        vtkSmartPointer<vtkPoints> inPts = isosurface_mesh->GetPoints();
+        vtkIdType numPts = inPts->GetNumberOfPoints();
+
+        std::vector<vital::vector_3d> points(numPts);
+        std::vector<vital::rgb_color> colors;
+
+        for (vtkIdType i = 0; i < numPts; ++i)
+        {
+          inPts->GetPoint(i, points[i].data());
+        }
+
+        auto pointcloud_writer = new kwiver::arrows::pdal::pointcloud_io();
```

Comment:
This code should be in an ifdef checking if PDAL is enabled. Log and error and return false if it's not enabled.